src/analytics.ts

```diff
@@ -12,15 +12,19 @@ type Options = {
 export function createAnalyticsInstance(options?: Options) {
     let _growthbook: Growthbook, _rudderstack: RudderStack
 
-    const initialise = ({ growthbookKey, growthbookDecryptionKey, rudderstackKey, enableDevMode }: Options) => {
-        _rudderstack = RudderStack.getRudderStackInstance(rudderstackKey)
-        if (growthbookKey && growthbookDecryptionKey) {
-            _growthbook = Growthbook.getGrowthBookInstance(growthbookKey, growthbookDecryptionKey, enableDevMode)
-        }
+    const initialise = () => {
+        _rudderstack = RudderStack.getRudderStackInstance(
+            ['.pages.dev', 'git-fork', 'localhost', 'staging'].some(condition =>
+                window.location.hostname.includes(condition)
+            )
+                ? '1lN3tsFD2nruGFgM5F074DC2hMB'
+                : '1oV5agvlcnCZ6IH94wCKM1oR8Pd'
```

Comment:
Why are we hardcoding the rudderstack keys here ?

Reply:
discussed in thread: https://deriv-group.slack.com/archives/C05DUEXQLU8/p1701158017753789